lambdas/sharedattributes/src/test/java/uk/gov/di/ipv/core/sharedattributes/SharedAttributesHandlerTest.java

```diff
@@ -217,9 +244,9 @@ void shouldReturn500WhenUnableToSignSharedAttributes() throws JsonProcessingExce
                 .thenReturn(
                         Map.of(
                                 "CredentialIssuer1",
-                                objectMapper.writeValueAsString(CREDENTIAL_INPUT_1),
+                                objectMapper.writeValueAsString(CREDENTIAL_INPUT_2),
                                 "CredentialIssuer2",
-                                objectMapper.writeValueAsString(CREDENTIAL_INPUT_2)));
+                                objectMapper.writeValueAsString(CREDENTIAL_INPUT_1)));
```

Comment:
These keys now don't match with the names of the values? Can they be re-ordered to match?